testsuite/features/srv_delete_channel_npn.feature

```diff
@@ -0,0 +1,79 @@
+# COPYRIGHT (c) 2017 SUSE LLC
+# License under the terms of the MIT License.
+
+Feature: delete channels
+
+  Scenario: Clone the first channel
+    Given I am on the manage software channels page
+    When I follow "Clone Channel"
+    And I select "Test-Channel-x86_64" as the origin channel
+    And I click on "Clone Channel"
+    And I should see a "Create Software Channel" text
+    And I should see a "Current state of the channel" text
+    And I click on "Clone Channel"
+    Then I should see a "Clone of Test-Channel-x86_64" text
+
+  Scenario: Clone the second channel using first channel as base
+    Given I am on the manage software channels page
+    When I follow "Clone Channel"
+    And I select "Clone of Test-Channel-x86_64" as the origin channel
+    And I click on "Clone Channel"
+    And I should see a "Create Software Channel" text
+    And I should see a "Current state of the channel" text
+    And I click on "Clone Channel"
+    Then I should see a "Clone of Clone of Test-Channel-x86_64" text
+
+  Scenario: Try delete channel with clone
+    Given I am on the manage software channels page
+    When I follow "Clone of Test-Channel-x86_64"
+    And I follow "Delete software channel"
+    And I check "unsubscribeSystems"
+    And I click on "Delete Channel"
+    Then I should see a "Clone of Test-Channel-x86_64" text
+    And I should see a "Unable to delete channel" text
+
+  Scenario: Delete channel without clones neither children
+    Given I am on the manage software channels page
+    When I follow "Clone of Clone of Test-Channel-x86_64"
+    And I follow "Delete software channel"
+    And I check "unsubscribeSystems"
+    And I click on "Delete Channel"
+    Then I should see a "Clone of Clone of Test-Channel-x86_64" text
+    And I should see a "has been deleted." text
```

Comment:
I would avoid the dot, just in case that sentence change in the feature to something longer. But is perfectly fine as it is now.

Reply:
thanks oscar. I follow your suggestion and just remove the dot